examples/nas/oneshot/spos/architecture_final.json

```diff
@@ -1,22 +1,22 @@
 {
-  "LayerChoice1": [false, false, true, false],
-  "LayerChoice2": [false, true, false, false],
-  "LayerChoice3": [true, false, false, false],
-  "LayerChoice4": [false, true, false, false],
-  "LayerChoice5": [false, false, true, false],
-  "LayerChoice6": [true, false, false, false],
-  "LayerChoice7": [false, false, true, false],
-  "LayerChoice8": [true, false, false, false],
-  "LayerChoice9": [false, false, true, false],
-  "LayerChoice10": [true, false, false, false],
-  "LayerChoice11": [false, false, true, false],
-  "LayerChoice12": [false, false, false, true],
-  "LayerChoice13": [true, false, false, false],
-  "LayerChoice14": [true, false, false, false],
-  "LayerChoice15": [true, false, false, false],
-  "LayerChoice16": [true, false, false, false],
-  "LayerChoice17": [false, false, false, true],
-  "LayerChoice18": [false, false, true, false],
-  "LayerChoice19": [false, false, false, true],
-  "LayerChoice20": [false, false, false, true]
-}
+    "_mutation_1": "2", 
```

Comment:
Suggest use customized labels (`label="my_own_label"`) as the default label format has changed in the latest version

Reply:
I have changed all label as `"default_xxx"`, as in our implementation of supernet the labels of layerchoice name like that.

Reply:
Have you tested it? This doesn't sound correct to me.
Why is `LayerChoice(..., label='my_own_label')` not favored?

Reply:
Have updated the script with setting label as "LayerChoice1".